src/richText.js

```diff
@@ -145,8 +113,17 @@ define([
      * if the editor does not exist.
      *
      * @param input - editor jQuery HTML element.
-     * @param form - form object; used for creating bubbles.
-     * @param options - rich text bubble options.
+     * @param form - form object, functions used:
```

Comment:
This could have a better name. It's really just a collection of required methods to pass in to the editor. All of them except `getIconFromPath` could have sensible defaults derived from a js-xpath instance